packages/backend/src/config/app-env-vars/formsg.ts

```diff
@@ -0,0 +1,14 @@
+export const formSgConfig = Object.freeze({
```

Comment:
```suggestion
export const formsgConfig = Object.freeze({
```
same thing for consistency sake if you decide to change the naming

Reply:
good call, done!